survey-sdk/src/main/java/co/nimblehq/survey/sdk/request/LoginRequest.kt

```diff
@@ -0,0 +1,15 @@
+package co.nimblehq.survey.sdk.request
+
+import com.squareup.moshi.Json
+
+data class LoginRequest(
+    @Json(name = "grant_type")
+    val grantType: String,
+
+    @Json(name = "email")
+    val email: String,
+
+    @Json(name = "password")
+    val password: String,
+
+    ) : BaseRequest()
```

Comment:
the formatting is quite weird here
```suggestion
data class LoginRequest(
    @Json(name = "grant_type")
    val grantType: String,

    @Json(name = "email")
    val email: String,

    @Json(name = "password")
    val password: String,

) : BaseRequest()
```

Reply:
@luongvo Could you please give me more detail?

Reply:
@doannimble you can get the point from my suggestion, right?
![image](https://user-images.githubusercontent.com/16315358/134038995-cbca5c69-34ed-4583-ad68-9368d934e79f.png)

Reply:
@luongvo Ah yes, there is something wrong with the format style in AS. I'm trying to figure out.

Reply:
@luongvo Updated